Manifest.toml

```diff
@@ -83,7 +83,7 @@ version = "0.10.3"
 
 [[IRTools]]
 deps = ["InteractiveUtils", "MacroTools", "Test"]
-git-tree-sha1 = "09b9d429fd57ad899d579dbe6d0f68efd82e9ba4"
+git-tree-sha1 = "4d0c6647aa89b3678d03967ca11c8a5cdc83d2b5"
```

Comment:
It would be best not to combine this with an IRTools version change, unless it's strictly needed for tests to pass.

Reply:
That must've happened by accident, I undid it.